docs/product-user-guide/src/main/asciidoc/guided-rules-WHEN-proc.adoc

```diff
@@ -1,50 +1,54 @@
-+
-Once you select a restriction, the dialog window closes automatically.
-+
-.Modifying a Condition
-image::5689.png[]
-
-. Choose an operator for the restriction (for example `greater than`) from the drop-down menu next to the added restriction.
-
-. Click *Edit* (image:6191.png[]) to define the field value. The field value can be a literal value, a template key, a formula, or an expression editor.
-
-. To apply multiple field constraint, click the condition and in the *Modify constraints for LoanApplication* dialog window, select *All of(And)* or *Any of(Or)* from the *Multiple field constraint* drop-down menu.
-+
-.Adding Multiple Field Constraint
-image::5688.png[]
-+
-The constraint is displayed in the Guided Rule Template Editor.
-
-. Click the constraint.
-+
-The *Add fields to this constraint* dialog window opens.
-
-. Specify the fields and their values. For example, by clicking *New Formula*, you can add a new formula expression that is later evaluated to be true or false.
-+
-.Multiple Field Constraint on Formula Expression
-image::loanapp.png[]
+. In the guided rule designer, click the plus icon (image:5686.png[]) on the right side of the `WHEN` section.
++
+The *Add a condition to the rule* window with the available condition elements opens.
++
+.Add a condition to the rule
+image::5687.png[Add a condition to the rule]
++
+The list includes the data objects from the *Data Objects* tab of the guided rule designer, any DSL objects defined for the package (if you selected *Use Domain Specific Language (DSL)* when you created this guided rule), and the following standard options:
+
+* *The following does not exist:* Use this to specify facts and constraints that must not exist.
+* *The following exists:* Use this to specify facts and constraints that must exist. This option is triggered on only the first match, not subsequent matches.
+* *Any of the following are true:* Use this to list facts and constraints that must be true.
+* *From:* Use this to define a `From` conditional element for the rule.
+* *From Accumulate:* Use this to define an `Accumulate` conditional element for the rule.
+* *From Collect:* Use this to define a `Collect` conditional element for the rule.
+* *From Entry Point:* Use this to define an `Entry Point` for the pattern.
+* *Free form DRL:* Use this to insert a free-form DRL field where you can define condition elements freely, without the guided rule designer.
++
+*<@Michael (1 of 6): I was going to put a note here about template keys, but they don't apply to free form DRL within the guided rule designer, right? Or do they? Since I don't see any option in the guided editor either for template keys, when I modify values, I assume it's safe to say that template keys just don't apply to guided rules, only to guided decision tables, guide rule templates, and decision tables, where you can define the values to be interpolated with the keys. Maybe DRL rules too, but I'm not there yet. That right?>*
```

Comment:
Correct! They only apply to Guided Rule Templates and Guided Decision Table BRL Fragments.

XLS(X) decision tables do not support ```@{key}``` syntax for place-holders but ```$param``` etc.